### ISSUE TITLE
Open source ADPixci repository

### ISSUE THREAD

**Issue by irierr** [CentralLaserFacility/ADPixci#82]
The software controls team has a goal of open sourcing a portion of our work, so that it can be used by the wider scientific community. 
As an initial step, we will investigate open sourcing the CentralLaserFacility/ADPixci and repository and implement changes to ready it for open sourcing.
It can then be used as an example for future work.

**Comment by irierr** [comment]
### Notes for Improving Documentation
- ADPixci module uses a PCI to CameraLink interface
- [ADPixci architecture diagram](https://stfc365.sharepoint.com/:p:/r/sites/CLFSoftwareControlsTeam/Shared%20Documents/Raptor%20Camera/Documentation/ADPixci_Architecture.pptx?d=w4080ada47e7546d39612d2de6bdbf773&csf=1&web=1&e=2KP0KD)

**Comment by irierr** [comment]
Currently working on #94, #126.
Still need to do #127, #122, #98.
Waiting on reviews for #128, #130, [raptor-eagleXV#54](https://github.com/CentralLaserFacility/raptor-eagleXV/issues/54).

**Comment by paul-richardson-clf** [comment]
Scrum 29/5:
By 6th/6: do #94, get reviews finished. Test against Lite SDK (#98) - see consequences
aim to submit #126 form by Fri

Scrum 12/6:
#94 still in progress, but revealed further errors - aim to complete on Tue. Not started #98. #126 in progress (IP form).
#127 to be merged

Scrum 17/6:
Concentrating on.
#94 is revealing new bugs, esp. related to parameter setting. Also looking to check thread safe as 3 threads running.
#98 will remove if doesn't work.

Scrum 24/6:
#94 init taking from 30 seconds to few minutes, investigating cache, buffers etc. Have a few ideas to pursue.

Scrum 3/7
Fixed slow init issue. Blocked on memory leak tool (IT).

**Comment by irierr** [comment]
#94: still working through bugs. 
#127: will be merged after #94.
#98, #122: not started
#126: waiting for other tasks to be merged before submission.

**Comment by irierr** [comment]
#94, #98, #127 waiting to be merged
#122 will be started tomorrow
#126 waiting for all other tasks to complete
